stock_customer_deposits/readme/USAGE.md

```diff
@@ -0,0 +1,38 @@
+To use this module, you need to:
+
+Create Customer deposits
+
+1. Go to Sales > Quotations.
+2. Create new quotation to the desired customer
+3. Add storable product in order lines.
+4. Select route **Customer Deposit** in line.
+5. Add quantity you will keep as a deposit.
+6. Confirm quotation.
+7. Two pickings will be created.
```

Comment:
**suggestion:** Please explain _why_, what's the purpose of each of those pickings.